### DISCUSSION TITLE
Tibber

### DISCUSSION THREAD

**Discussion by meijer3** [mampfes/ha_epex_spot#67] [General]
Hi all, I live in Sweden (SE4) and I have Tibber. 
I have two questions, hopefully someone can help me.

1. The prices are in EUR, is it possible to add other currencies? This happens more often in other parts in HA, so I have a sensor `sensor.eur2sek`. Maybe an idea to link this script somehow to an other sensor/number?

2. I do understand that Tibber has no offical fix markup, ony VAT. But they do, and i believe they are dynamic? How can we add this into the config? And I cannot find them anywere...

Keep up the good work!

**Comment by mampfes** [comment]
1. The prices for SEx are traded in EUR, right? How would you convert from EUR to SEK?
2. Sorry, I didn't understand the question. Dynamic taxes/surcharges are not supported.

**Comment by meijer3** [comment]
1. Yes I think they all trade in EUR. I do have a sensor that keeps track of my currency
![image](https://github.com/mampfes/ha_epex_spot/assets/45625341/a99cff9d-b9d6-4fc6-b79d-e2d1d6cd0ec9)
So either in the config of the add-on, I would point to this sensor? Or I only do it in the card, but I pref the first option.

2. I was looking for some Swede that already figured out, howmucht tibber is adding, so I could calculate it my self. 

thanks for your time!

**Reply by meijer3** [reply]
I fixed my currency in the fronend.
Not sure what tibber taxes on top of the bruto hourly price, but the bruto is now the same as the tibber app atleast

```js
    data_generator: |
      return entity.attributes.data
      .map((entry) => {
        const curr = temp1.states['sensor.eur2sek_2'].state;
        const v =  entry.price_ct_per_kwh * curr
        return [new Date(entry.start_time), v ];
      });
```
